fit_resonator/check_data.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+import os
+
+changes_made = False
+filepath = ''
+delim = ","
+
+
+def file(filename):
+    # read in file
+    global filepath
```

Comment:
Do other functions in this file read or write `filepath`? Does it need to be global or can it be a static variable like in C (there are some hacks in Python to implement static variables that are updated every time a function is called)? Or could `file()`, `raw()`, and `parse()` be part of a simple class that have `filepath` and `changes_made` as class members?